CHANGELOG.md

```diff
@@ -8,5 +8,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-* 
-* Set up project and CI/CD GitHub actions, [PR-9](https://github.com/reductstore/reduct-go/pull/9)
\ No newline at end of file
+* Set up project and CI/CD GitHub actions, [PR-9](https://github.com/reductstore/reduct-go/pull/9)
+## Added Bucket api [PR-10](https://github.com/reductstore/reduct-go/pull/10)
```

Comment:
Just the  title of the PR and the link to it:

```
* Implement Bucket API [PR-10](https://github.com/reductstore/reduct-go/pull/10)
```

You should add the list of all changes to the PR description . There was a template which you deleted ....